process/css-styleguide/modal/modal.css

```diff
@@ -0,0 +1,11 @@
+@import '../default.css';
+
+body {
+  background-color: #fbfbfb;
+}

```

Comment:
The modal shouldn't change the background-color of the page. What if I want to use it on the landing page which has a `valhalla` background-color? This is gonna break the page style and it is not what we want.